Raclette/levels/victory.gd

```diff
@@ -0,0 +1,29 @@
+tool
+extends Control
+
+onready var _anim: = $AnimationPlayer
+onready var _hider = $CanvasLayer/Hider
+
+export(String, FILE) var title_screen_path: String
+onready var _title_screen: = load(title_screen_path)
+
+
+func _ready() -> void:
+	_anim.play("fade_in")
+	yield(_anim, "animation_finished")
+	_hider.queue_free()
+
+
+func _on_TitleScreen_pressed() -> void:
+	get_tree().change_scene_to(_title_screen)
+
+
+func _on_Quit_pressed() -> void:
+	get_tree().quit()
+
+
+func _get_configuration_warning() -> String:
```

Comment:
Unconsistent naming format, should be _get_Configuration_warning

Reply:
Not this one. The previous functions (_on_Retry_pressed for instance) reference a Node with its name, which is in CamelCase so I suppose it is alright to keep the exact name just for this. The convention is snake_case for functions but this case might be an exception

Reply:
Oh! Yeah alright, did not saw this (-: